src/asyncAwait.js

```diff
@@ -0,0 +1,29 @@
+const fetch = require('node-fetch')
+const getContributorsA = async (orgainzation) => {
+  let repos = await fetch(`https://api.github.com/orgs/${orgainzation}/repos`).catch(err => console.error(err))
+  let status = await repos.status
+  if (status === 403) {
+    let headers = await repos.headers
+    let header = headers[Object.getOwnPropertySymbols(headers)[0]]
+    let date = await new Date(header['x-ratelimit-reset'] * 1000)
+    return `Limit: ${header['x-ratelimit-remaining']} left of ${header['x-ratelimit-limit']}. Wait until ${date.toTimeString()}`
+  }
+  let reposJson = await repos.json()
+  return contributorsPerRepo(reposJson, orgainzation)
+}
+
+const contributorsPerRepo = async (repoList, orgainzation) => {
+  return repoList.filter((item, key) => { if (key < 3) { return item } }).reduce(async (collection, current) => {
```

Comment:
there exists better way to get first 3 elements of an array than `filter` in such way.
also `3` looks like magic number here, what does it mean?